src/components/Inventory.js

```diff
@@ -1,29 +1,43 @@
 import React from "react";
 import DataTable from "./DataTable";
-
-// import {
-//   useTable,
-//   useGroupBy,
-//   useFilters,
-//   useSortBy,
-//   useExpanded,
-//   usePagination
-// } from "react-table";
+const axios = require("axios");
 
 class Inventory extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      output: []
+    };
+  }
+
+  async componentDidMount() {
+    var that = this;
+    axios
+      .get("http://localhost:2999/inventory")
+      .then(function(response) {
+        that.setState({ output: response.data });
+        console.log(response);
+      })
+      .catch(function(error) {
+        console.log(error);
+      });
+  }
+
   render() {
-    const headings = ["Item", "Count", "Price", "Cost", "Profit"];
+    if (this.state.output !== []) {
+      const headings = ["Item", "Count", "Price", "Sold"];
+      console.log(this.state.output);
+      const api_rows = this.state.output;
```

Comment:
rename the state variable to be api_rows, and then pass in rows = {this.state.api_rows}